### PR TITLE
Prepare v1.18.0 release

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Update changelog and docs.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [x] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,87 @@
...
 
+## [v1.18.0](https://github.com/FerretDB/FerretDB/releases/tag/v1.18.0) (2024-01-08)
+
+### What's Changed
+
+#### Capped collections
+
+This release adds support for capped collections.
+They can be created as usual using `create` command.
+Both `max` (maximum number of documents) and `size` (maximum collection size in bytes) arguparametersments are supported.
```
Comment (henvic) [inline CHANGELOG.md:13]:
```suggestion
They can be created as usual using the `create` command.
Both `max` (maximum number of documents) and `size` (maximum collection size in bytes) arguments are supported.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,87 @@
...
+This release adds support for tailable cursors.
+Both `tailable` and `awaitData` parameters are supported.
+
+#### OpLog tailing
+
+This release adds support for the basic OpLog functionality.
+The main supported use case is Meteor's OpLog tailing.
+Replication is not supported yet.
+
+It is not enabled by default; to enable it, create a capped collection `oplog.rs` in the `local` database.
```
Comment (henvic) [inline CHANGELOG.md:26]: Maybe this helps clarity:

```suggestion
OpLog tailing is not enabled by default; to enable it, create a capped collection `oplog.rs` in the `local` database.
```

File: CHANGELOG.md (hunk 1 above)
Comment (AlekSi) [reply CHANGELOG.md:13]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (AlekSi) [reply CHANGELOG.md]: Rewrote the whole sentence (technically, it is not even "OpLog _tailing_")

**Review by henvic — APPROVED** [review]
LGTM

**Review by Fashander — APPROVED** [review]

**Pull request merged**